test/integration/notifications.test.js

```diff
@@ -198,6 +198,7 @@ describe('Integration: notifications', () => {
           Accept: 'application/vnd.github.html+json',
         },
       }).get('/repos/github-slack/app/issues/31').reply(200, fixtures.issue);
+      nock('https://api.github.com').get('/repos/github-slack/app/pulls/31/reviews').reply(200);
```

Comment:
I think we want to always return a fixture with these stubbed out HTTP requests, similar to the line above. `fixtures.review` could look something like your example from `test/messages/pull-request.test.js`:
```js
[
        { state: 'CHANGES_REQUESTED', user: { login: 'user2' } },
        { state: 'APPROVED', user: { login: 'user2' } },
      ],
```